app/src/main/java/com/example/coinage/fragments/TransactionListFragment.java

```diff
@@ -58,9 +65,38 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         // populate recycler view with transactions
         queryTransactions();
 
+        tvTotalSpending = view.findViewById(R.id.tvDateDetail);
+        totalSpending();
+
         // (placeholder button, will replace with some representation of overall spending limit)
-        button2 = view.findViewById(R.id.button2);
-        button2.setOnClickListener((View v) -> goSpendingLimitOverview());
+        cardOverview = view.findViewById(R.id.cardOverview);
+        cardOverview.setOnClickListener((View v) -> goSpendingLimitOverview());
+    }
+
+    @Override
+    public void onResume() {
+        super.onResume();
+        // query again once returned to this fragment
+        queryTransactions();
+    }
+
+    private void totalSpending() {
```

Comment:
I think this method could have a clearer name - `totalSpending` would make sense as a variable name, but this looks like it could be called `fetchAndUpdateTotalSpending`